src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -840,6 +840,8 @@ private void BuildPurchaseOrderReqs(ModelBuilder builder)
             entity.HasOne(e => e.SecondAuthBy).WithMany().HasForeignKey(x => x.SecondAuthById);
             entity.Property(e => e.Email).HasColumnName("EMAIL_ADDRESS").HasMaxLength(50);
             entity.Property(e => e.InternalNotes).HasColumnName("INTERNAL_ONLY_ORDER_NOTES").HasMaxLength(300);
+            entity.Property(e => e.Description).HasColumnName("DESCRIPTION").HasMaxLength(2000);
+            entity.HasOne(e => e.ReqState).WithMany().HasForeignKey(e => e.State);
```

Comment:
I'm late with this sorry, but Description is already mapped above as PartDescription - if there's a part then description gets populated with the part.description when making the req so seemed appropriate. State is already mapped as a string too but not sure if you need some other part of the reqState object!

Reply:
I think the req description is a seperate thing to the PartDescription, so probably makes more sense to stick with Description since it might be different to the Part's description

Reply:
The old form had it labelled as part description on the front end but I guess if it's Sundry it's kind of not a part that's being described. Either probably works but whoever of us does a pr next should probably remove the duplicate string mapping to the same field and just keep one
![image](https://user-images.githubusercontent.com/51612225/160647632-7bddcf64-8087-455f-98b2-5edcd2f5b6e3.png)